types/vote_set_test.go

```diff
@@ -2,6 +2,8 @@ package types
 
 import (
 	"bytes"
+	"crypto/sha256"
+	"fmt"
```

Comment:
I think the IDE messed up and didn't auto run gofmt, as this "fmt" is unused and should be removed.